projects/plugins/jetpack/extensions/blocks/nextdoor/edit.js

```diff
@@ -93,23 +94,49 @@ export function NextdoorEdit( {
 		</Placeholder>
 	);
 
-	const blockPreview = () => {
-		return (
-			<>
-				<div className={ `${ defaultClassName }-overlay` }></div>
-				<iframe width="100%" height="200" frameBorder="0" src={ iframeUrl } title="Nextdoor" />
-			</>
-		);
-	};
-
 	return (
 		<div className={ className }>
 			<NextdoorControls
 				{ ...{ defaultClassName, nextdoorShareUrl, onFormSubmit, setNextdoorShareUrl } }
 			/>
-			{ iframeUrl ? blockPreview() : blockPlaceholder }
+			{ iframeUrl ? (
+				<BlockPreview
+					defaultClassName={ defaultClassName }
+					iframeUrl={ iframeUrl }
+					clientId={ clientId }
+				/>
+			) : (
+				blockPlaceholder
+			) }
 		</div>
 	);
 }
 
+const BlockPreview = ( { defaultClassName, iframeUrl, clientId } ) => {
+	const html = `
+		<script>
+			window.addEventListener( 'message', function ( event ) {
+				if ( ! event.origin.startsWith( 'https://nextdoor' ) ) {
+					return;
+				}
+				var iframe = document.getElementById( 'nextdoor-embed-${ clientId }' );
+				if ( event.source !== iframe.contentWindow ) {
+					return;
+				}
+				iframe.setAttribute( 'height', event.data.height );
+				var clientBoundingRect = document.body.getBoundingClientRect();
+				window.parent.postMessage( { action: 'resize', width: clientBoundingRect.width, height: clientBoundingRect.height }, '*' );
+			} );
+		</script>
+		<iframe id="nextdoor-embed-${ clientId }" width="100%" height="200" frameBorder="0" src="${ iframeUrl }" title="Nextdoor" />
```

Comment:
Should the `title` attribute be translated?

Reply:
Translated here https://github.com/Automattic/jetpack/pull/33931/commits/264f273f74e070b81bd98ec630a33aac66527cb9